encord/project.py

```diff
@@ -262,6 +262,8 @@ def submit_label_row_for_review(self, uid: str):
         """
         Submit a label row for review.
 
+        **Note:** this method is not supported for the workflow-based projects. See the documentation about the workflows
```

Comment:
Note: This method is not supported for workflow-based projects. Please see our workflow documentation for more details.

Again, if we can hyperlink that would be great. If not, don't worry :)